dotcom-rendering/src/web/layouts/ImmersiveLayout.tsx

```diff
@@ -338,7 +338,13 @@ export const ImmersiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 										ArticleSpecial.Labs ? (
 											<GuardianLabsLines />
 										) : (
-											<DecideLines format={format} />
+											<DecideLines
+												format={format}
+												color={
+													decidePalette(format).border
+														.article
+												}
+											/>
```

Comment:
If a palette can be inferred from a format, I would probably simplify this to simply the format prop and handle the colour within the component. Probably best as part of a subsequent refactor, though!

Reply:
Thanks, adding this to my list!